Assets/MixedRealityToolkit/_Core/Devices/OpenVR/OpenVRDeviceManager.cs

```diff
@@ -140,12 +143,10 @@ private static SupportedControllerType CurrentControllerType
                         case "Vive Knuckles - Left":                                            // TODO: Yet to test
                         case "Vive Knuckles- Right":                                            // TODO: Yet to test
                             return SupportedControllerType.ViveKnuckles;
-                        default:
-                            break;
                     }
                 }
 
-                return SupportedControllerType.GenericOpenVR;
+                return SupportedControllerType.None;
```

Comment:
why change this to None, it makes no sense?  The generic is the Fallback device, so any new OpenVR controller will be supported by default using the default options until it's added.

Please revert this.

Reply:
Bc any controller would be an open VR controller, not just open VR controller. 

I tried to get Xbox controller setup, and it came out as open vr.

Reply:
then you need to add an Xbox controller class to handle it differently, not default none as that breaks all other new controllers we haven't defined

Reply:
It doesn't break them, but just warns that they're unsupported.

Reply:
WinMR headset with OpenVR:

![image](https://user-images.githubusercontent.com/3580640/43618630-56c9ff4a-967e-11e8-96e2-53ef291c487a.png)

Previously this correctly fell back to `GenericOpenVR`.

Reply:
That's because it's not in the list, and appropriately shows that it's unsupported

Reply:
Agreed, but then how do we get around fact that even non open VR controllers get added?

Reply:
And it's not like we're hard coding anything, but adding the name of the controller to the list

Reply:
>Agreed, but then how do we get around fact that even non open VR controllers get added?

I keep trying, but I've yet to see an Xbox controller trigger this code. Do you have repro steps?

>And it's not like we're hard coding anything, but adding the name of the controller to the list

Yes, a hard-coded list of strings.

Reply:
I've got an idea about how to solve this.

In the controller configuration we can add a text field for people to put the name of the controller whenever they set it to generic, and update this switch to fall back to generic when a match is found. But we should definitely keep this line as none.

Reply:
> Yes, a hard-coded list of strings

It doesn't have to be.

Reply:
Where would we get the controller names from?

Reply:
> In the controller configuration we can add a text field for people to put the name of the controller whenever they set it to generic, and update this switch to fall back to generic when a match is found. But we should definitely keep this line as none.

In short, from the user's controller configuration file.

Reply:
@keveleigh I went a head and reverted this for now. We can pick it back up in beta.